thundra/integrations/sqlalchemy.py

```diff
@@ -0,0 +1,149 @@
+import traceback
+import time
+import logging
+from thundra import config, constants
+from thundra.plugins.invocation import invocation_support
+from thundra.integrations.rdb_base import RdbBaseIntegration
+from thundra.opentracing.tracer import ThundraTracer
+
+from sqlalchemy.event import listen
+from sqlalchemy.engine.interfaces import ExecutionContext
+
+logger = logging.getLogger(__name__)
+
+
+class SqlAlchemyIntegration(RdbBaseIntegration):
+    CLASS_TYPE = 'sqlalchemy'
```

Comment:
I think it is better to define as `SQLAlchemy` since as far as I know it is used directly on the UI